crates/swc/tests/fixture/issues-2xxx/2679/output/index.js

```diff
@@ -1,4 +1,2 @@
-!function() {
-    var a = {};
-    a.b = 1, a = null, window.test = a;
-}();
+var a;
```

Comment:
This is leaking `a` into the global variable list. Is that ok?

Reply:
Yes, it's in es module.